services/order.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import Ticket, Order, User
```

Comment:
Avoid importing `User` directly. Use `get_user_model()` to ensure consistency with Django's user model configuration.